docs/en_us/platform_api/source/course_structure/overview.rst

```diff
@@ -0,0 +1,36 @@
+.. _EdX Platform Course Structure API Overview:
+
+################################################
+EdX Platform Course Structure API Overview
+################################################
+
+Use the edX Platform Course Structure API to view course details, including the
+blocks in the course and the course grading policy.
+
+The edX Platform Course Structure API uses Representational State Transfer
+(REST) design principles and supports the JavaScript Object Notation (JSON)
+data-interchange format. The REST API is simple and lightweight.
+
+You can use the edX Platform Course Structure API for web, desktop, and mobile
+applications.
+
+********************************************
+EdX Platform Course Structure API Version 0
+********************************************
+
+The edX Platform Course Structure API is currently at version 0. We plan on
+making significant enhancements to this API.
```

Comment:
I believe the Course Structure API is internal only -- i.e. students can't actually access it. @clintonb: Am I remembering that correctly? If so, we should really note that somewhere.